### PR TITLE
Filter present speaker search-value selector

### PR DESCRIPTION
**Pull request by MaximilianKrambach**
This will filter the search value selector for list of speakers; and add a configuration option to only allow 'present' speakers to speak.

### CODE REVIEW

File: openslides/agenda/models.py [hunk 1]
```diff
@@ -344,6 +344,10 @@ def add(self, user, item, skip_autoupdate=False):
             raise OpenSlidesError(f"{user} is already on the list of speakers.")
         if isinstance(user, AnonymousUser):
             raise OpenSlidesError("An anonymous user can not be on lists of speakers.")
+        if config["agenda_present_speakers_only"] and not user.is_present:
+            raise OpenSlidesError("Only present users can be on the lists of speakers.")
+        if self.filter(user=user, item=item, begin_time=None).exists():
```
Comment (FinnStutzenstein) [inline openslides/agenda/models.py]: This seems to be a rebase issue: Please remove these lines (they are 6 lines above)..

**Comment by MaximilianKrambach** [comment]
@emanuelschuetze: I  fixed the error

**Comment by FinnStutzenstein** [comment]
Please wait for #4639 to be merged.

**Comment by MaximilianKrambach** [comment]
Rebased and updated to #4639

**Pull request merged**